src/App.css

```diff
@@ -52,6 +53,44 @@ tr {
   text-align: center;
 }
 
+.cards {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
 .card {
-  width: 18rem;
+  width: 25rem;
+  margin: 2rem auto;
+  text-align: center;
+}
+
+.selected {
+  background-color: rgb(181, 196, 221);
+}
```

Comment:
Good use of CSS to add a highlight to the selected row, to make this even better, consider using a lighter colour, this is to improve accessibility, a dark color background with dark color text can make it hard for some people to read. Here is a good tool for making sure the contrast between text color and background color is sufficient for everyone to use: https://webaim.org/resources/contrastchecker/